CompilerSource/compiler/compile.cpp

```diff
@@ -160,14 +160,27 @@ dllexport void ide_handles_game_launch() { run_game = false; }
 static bool redirect_make = true;
 dllexport void log_make_to_console() { redirect_make = false; }
 
+bool ends_with(std::string const &fullString, std::string const &ending) {
```

Comment:
```suggestion
static bool ends_with(std::string const &fullString, std::string const &ending) {
```
`static` or anonymous namespace, please. Is there no utility for this?

Reply:
Ok: 966e032b196f337efc19b71e20f825bd5de1a926

Since you rebased master, I had to reapply this manually with another commit because apparently I can't just "Commit suggestion" again. I did a search and couldn't find a utility for checking ends with anywhere else in the entire repo. I think we should also add a `string_ends_with` to the engine later.